### PR TITLE
Add prometheus metrics

### PR DESCRIPTION
**Pull request by BenBagBag**
This PR adds basic Prometheus metrics to the Django app. There is plenty of room for expanding these metrics with custom rules, and for displaying them in Grafana or some other third party metrics app.

This is a draft PR at the moment, because we need to verify that this setup will work in production. I don't have enough information about the deploy/prod setup to do this. At a minimum, we will need to set up a Prometheus instance in the prod environment. 

Long-term, we will need to set up a Prometheus repo if we want to use custom rules (aka custom metrics). Custom rules are stored in a `rules.yml` file in the directory you run Prometheus from, so that should really be tracked in its own repo.

### CODE REVIEW

File: bankgreen/settings.py [hunk 1]
```diff
@@ -68,11 +68,13 @@
     "django_admin_listfilter_dropdown",
     "django_filters",
     "django_json_widget",
+    "django_prometheus",
```
Comment (brugo-dev) [inline bankgreen/settings.py:71]: Shouldn't this dependency be added to `requirements.txt`?

File: bankgreen/urls.py [hunk 2]
```diff
@@ -48,4 +48,5 @@
     path("sustainable-eco-banks/<str:tag>/", views.brand_redirect, name="brand_quicklink"),
     path("update_success/", views.update_success, name="update_success"),
     path("rest_api_view/", include("api.urls")),
+    re_path("", include("django_prometheus.urls")),
```
Comment (brugo-dev) [inline bankgreen/urls.py:51]: Is there a reason why you are using `re_path` here instead of `path`?

**Comment by brugo-dev** [comment]
The code style check failed on two files.  Make sure you clean your code by running `black .` from the top level of the repo before you push :)

**Review by brugo-dev — COMMENTED** [review]
Tested locally and it's working. 

I can't speak to the production set up, so Simon will have to answer those questions.

File: bankgreen/settings.py (hunk 1 above)
Comment (BenBagBag) [reply bankgreen/settings.py:71]: Yes, my bad.

File: requirements.txt
Comment (shuesken) [inline requirements.txt:1]: What's the deal with all these new requirements?

**Comment by BenBagBag** [comment]
@shuesken @brugo-dev Got my new machine working, tested this in and out of prod setting. Works fine locally without running Prometheus so long as your `.env` is set to local. Try giving it a go in actual prod.

**Comment by RogerTangos** [comment]
Closing this PR as it doesn't seem we need these at the moment

**Pull request closed without merging**